Homework - OOP 3 Phone/src/main/java/Agenda.java

```diff
@@ -0,0 +1,17 @@
+public class Agenda {
+
+    public void addToAgenta(Contact contact) {
+
+        System.out.println("A fost adaugat contactul " + contact);
+
+    }
+
+    public void makeACall(Contact contact){
```

Comment:
The functiononality: sendSMS, makeACall are phone functionality and not Agenta functionality:)
In the homework you do have the following code:
- phone.sendMessage("phone number", "message content");
  -     phone.listMessages("phone number");